boxsdk/object/legal_hold_policy.py

```diff
@@ -0,0 +1,80 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+
+import json
+
+from .base_object import BaseObject
+from boxsdk.util.translator import Translator
+from ..config import API
+from ..pagination.marker_based_object_collection import MarkerBasedObjectCollection
+
+class LegalHoldPolicy(BaseObject):
+    """Represents a Box legal_hold_policy"""
+
+    _item_type = 'legal_hold_policy'
+
+    def get_url(self, *args):
+        return self._session.get_url('legal_hold_policies', self._object_id, *args)
```

Comment:
Ha, is this the first object we have that isn't pluralized by just adding an "s" at the end?